ydb/core/viewer/viewer_topic_data.h

```diff
@@ -57,9 +57,11 @@ class TTopicData : public TViewerPipeClient {
 private:
     ui64 TabletId;
     TString TopicPath;
-    i64 PartitionId;
-    i64 Offset;
-    i64 Limit;
+    TMaybe<ui64> PartitionId;
+    TMaybe<ui64> Offset;
+    TMaybe<ui64> Timestamp;
+    TMaybe<ui64> Limit;
```

Comment:
```suggestion
    ui64 PartitionId = 0;
    ui64 Offset = 0;
    ui64 Timestamp = 0;
    ui64 Limit = 0;
```

much simplier, isn't it? :)